fuel/app/tests/api/v1.php

```diff
@@ -625,6 +625,68 @@ public function test_widget_publish_perms_verify(): void
 		$this->assertTrue($output);
 	}
 
+	public function test_participation_score_equals_input_value()
+	{
+		$this->markTestIncomplete(); // gotta make sure it ws made
+		return;
```

Comment:
This test is short circuited? intentional?

Reply:
Mmm... a little bit intentional? The rest of the test passed but I saw that the `test_session_play_create` method had these lines so I also put them here afterwards.

Reply:
I removed the short circuit and made sure the test passes, presumably it's working as intended now.